### ISSUE TITLE
php7.3-fpm needs nscd

### ISSUE THREAD

**Issue by hknet** [oerdnj/deb.sury.org#1876]
**Frequently asked questions**

- [x] I have read [Frequently Asked Questions](https://github.com/oerdnj/deb.sury.org/wiki/Frequently-Asked-Questions)
- [x] I have looked at the list of the existing issues (including closed issues) and searched if my issue has been already reported

**Describe the bug**
php7.3-fpm package on debian v11 (bullseye) somehow needs nscd to start but does not require it.

after removing nscd from the server the php7.3-fpm update fails to restart as it is looking for nscd

repo used: packages.sury.org

apt install php7.3-fpm results in:

Created symlink /etc/systemd/system/multi-user.target.wants/php7.3-fpm.service → /lib/systemd/system/php7.3-fpm.service.
insserv: FATAL: service nscd has to be enabled to use service php7.3-fpm
Failed to start php7.3-fpm.service: Unit nscd.service not found.
invoke-rc.d: initscript php7.3-fpm, action "start" failed.

basically - in my point of view - it should not even require nscd.

**Comment by oerdnj** [comment]
It **does not** require `nscd`. The question is what you did to your system that it does require it? And that's the question you can only answer yourself.
